.github/pull_request_template.md

```diff
@@ -9,16 +9,49 @@
 
 ## Issue type checklist
 <!--- What type of change are you submitting? Check the boxes that apply: -->
-- [ ] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
-- [ ] **New feature**: non-breaking change that adds functionality.
-- [ ] **Breaking change**: fix or feature that will cause current functionality to change.
+
+- [] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
+- [] **New feature**: non-breaking change that adds functionality.
+- [] **Breaking change**: fix or feature that will cause current functionality to change.
 
 ## Process & policy checklist
 <!--- Review the list and check the boxes that apply. -->
-- [ ] I have added tests for my changes.
-- [ ] I have tested my changes.
-- [ ] I have updated the project documentation to reflect my changes.
-- [ ] I have read the CONTRIBUTING documentation and followed the standards set for this project.
 
- 
-Click here for details on [formatting pull requests](https://microsoft.github.io/fast-dna/docs/contributing/working#formatting-messages).
+- [] I have added tests for my changes.
+- [] I have tested my changes.
+- [] I have updated the project documentation to reflect my changes.
+- [] I have read the CONTRIBUTING documentation and followed the standards set for this project.
+
+## Pull request formatting
+
+**`<type>`** is required to be at least one of the following:
+
+- break: update that changes the current functionality
+- chore: update that does not impact distributed packages
+- feat: add, update, or remove a feature
+- fix: remove property to resolve style issue
+
+**`<description>`** is required and speaks to what the user gets from this PR:
+
+- Be concise
+- Use lowercase
+- Use imperative, present tense (e.g. `add` not `adds`)
+- Do not end with a period. Period.
```

Comment:
Can we remove the second "Period.", it just sounds condescending.

Reply:
I would also remove the period from the last bullet